src/customGraphQLValidationRules.js

```diff
@@ -108,20 +108,21 @@ export function RequiredFields(context, options) {
     // know if that fragment covers all of the possible type options.
     Field(node) {
       const def = context.getFieldDef();
-
-      requiredFields.forEach(field => {
-        if (fieldAvailableOnType(def.type, field)) {
-          const fieldWasRequested = getFieldWasRequestedOnNode(node, field);
-          if (!fieldWasRequested) {
-            context.reportError(
-              new GraphQLError(
-                `'${field}' field required on '${node.name.value}'`,
-                [node]
-              )
-            );
+      if (def) {
```

Comment:
Rather than indent entire block to encapsulate it with this `if` conditional, would it be possible to opt for the behavior that was here before which `return`ed when `def` wasn't truthy?:

https://github.com/apollographql/eslint-plugin-graphql/blob/f1331d2992488ff9ae795f0625f1c8c6001f3a73/src/customGraphQLValidationRules.js#L34-L36

(In general, reducing over-indentation seems like a good principle to maintain.)